policyengine_us/parameters/gov/states/hi/tax/income/subtractions.yaml

```diff
@@ -0,0 +1,19 @@
+description: Hawaii counts these subtractions from adjusted gross income.
+values:
+  2021-01-01:
+    - pension_income # Line 13
+    - taxable_social_security # Line 14
+    - hi_military_pay_exclusion # Line 15
+    - us_govt_interest # Line 18
+    - student_loan_interest # Line 18
```

Comment:
both line 18?

Reply:
Yes, Line 18 is 'Other Hawaii Subtractions From Federal AGI', including several interests that can be subtracted

Reply:
> both line 18?
Yes, Line 18 is 'Other Hawaii Subtractions From Federal AGI', including several interests that can be subtracted